lib/node_modules/@stdlib/string/len/benchmark/benchmark.js

```diff
@@ -0,0 +1,50 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var fromCodePoint = require( '@stdlib/string/from-code-point' );
+var UNICODE_MAX = require( '@stdlib/constants/string/unicode-max' );
+var randu = require( '@stdlib/random/base/randu' );
+var floor = require( '@stdlib/math/base/special/floor' );
+var pkg = require( './../package.json' ).name;
+var len = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var str;
+	var out;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		str = fromCodePoint( floor( randu() * UNICODE_MAX ) ) + 'eep boop';
+		out = len( str );
+		if ( out !== 9 ) {
```

Comment:
I think you meant that that condition will always be false and the if block won't be entered.
Isn't that the point? If it evaluates to true, then the benchmark would fail, right? I am not sure I understand. For example, wouldn't [this condition](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/string/from-code-point/benchmark/benchmark.js#L50) always evaluate to false?

Reply:
@ShraddheyaS Yes, sorry. I misspoke. I meant that `out` will always equal `9`, even if the generated character is half of a surrogate pair.

Reply:
So that means I don't need to do anything here, right?